bucket/charm-gum.json

```
@@ -0,0 +1,44 @@
+{
+    "version": "0.8.0",
+    "homepage": "https://charm.sh/",

```

```suggestion
    "homepage": "https://github.com/charmbracelet/gum",
```
This is a more accurate homepage for gum